web-api/src/config/load_config.rs

```diff
@@ -12,7 +12,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
```

Comment:
:art: since this module is now a submodule of `config` we could simplify the name to `load` such that we import from `crate::config::load::*` instead of `crate::config::load_config::*`.

Reply:
I actually wanted to fold it into `config.rs` as the methods in there are just thin wrappers around this, but didn't do that so that github shows it as moved instead of removed+added. I will do so now.

Reply:
maybe now rename the parent into `load_config.rs`?? Idk.